.github/workflows/pr-preview.yml

```diff
@@ -0,0 +1,36 @@
+name: PR-Preview
+
+on:
+  pull_request:
+    branches: [ main ]
+
+env:
+  REACT_APP_NETWORK_IDS: "100,10200"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.14.2
```

Comment:
I wonder if there's an easy way to take this from the [`engines.node` key in `package.json`](https://github.com/gnosischain/gbc-deposit-ui/blob/bc5d33d6924be94d3652e227360fe8c7a6ded069/package.json#L5).

Reply:
now solved 🙌 we get the version automatically from package.json